app/viewmodels/PreviousDocumentType.scala

```diff
@@ -24,9 +24,9 @@ final case class PreviousDocumentType(
   previousAdminReference: PreviousAdministrativeReference
 ) {
 
-  val display: String = if (documentType.description.nonEmpty) {
+  val display: String = if (documentType.description.exists(_.nonEmpty)) {
```

Comment:
If the description is nonEmpty then it must exist. Is there a need to change that line?

Reply:
@kenneth-rayner since `description` is now optional, in order to maintain the same functionality, we need to return `false` if the description is `None` or `Some("")` and `true` if it is some non-empty string

Reply:
Calling `.nonEmpty` on a `String` checks whether the string is **empty** or not. Calling `.nonEmpty` on an `Option` checks whether the string is **defined** or not. i.e. `"".nonEmpty = false` but `Some("").nonEmpty = true`. I'll add some unit tests to ensure it's covered.

Reply:
@kenneth-rayner unit tests added - https://github.com/hmrc/transit-movements-trader-manage-documents/pull/56/commits/d09627d5ec56fe4d7151544e2d19d34486492701